content/en/platform/corda/5.0/developing-applications/getting-started/utxo-advanced-ledger-extensions/ledger-extensions-api-reference.md

```diff
@@ -0,0 +1,349 @@
+---
+date: '2023-06-01'
+title: "Ledger Extensions API Reference"
+project: corda
+version: 'Corda 5.0'
+menu:
+  corda5:
+    identifier: corda5-utxo-ledger-extensions-api-reference
+    parent: corda5-utxo-advanced-ledger-extensions
+    weight: 4600
+section_menu: corda5
+---
+
+# Advanced Contract Design
```

Comment:
The heading weights still aren't right - the "Advanced Contract Design" paragraph is a not on the following, it's not the heading for the whole page.
There should be another heading like "Advanced Ledger Types" or something.

Reply:
Updated to read:

# Ledger Extensions API Reference

## Advanced Contract Design

...

## Advanced Ledger Types

### Base API